src/util/decode_encode_muxed_account.js

```diff
@@ -4,28 +4,17 @@ import xdr from '../generated/stellar-xdr_generated';
 import { StrKey } from '../strkey';
 
 /**
- * Converts a Stellar address (in G... or M... form) to an XDR MuxedAccount
- * structure, forcing the ed25519 representation by default.
+ * Converts a Stellar address (in G... or M... form) to an `xdr.MuxedAccount`
+ * structure, using the ed25519 representation when possible.
  *
- * This optionally (that is, opt-in only) supports proper muxed accounts, where
- * an M... address will resolve to both its underlying G... address and an ID.
- * Note that this behaviour will eventually be the default.
- *
- * @function
- *
- * @param   {string}  address         G... or M... address to encode into XDR
- * @param   {bool}   [supportMuxing]  allows decoding of the muxed
- *     representation of the address, extracting the underlying ID from the M...
- *     address
+ * This supports full muxed accounts by default, where an M... address will
+ * resolve to both its underlying G... address and an ID.
```

Comment:
💡 Since it isn't configurable it is maybe misleading to say it is the default as if it is configurable.
```suggestion
 * This supports full muxed accounts, where an M... address will
 * resolve to both its underlying G... address and an ID.
```

Reply:
Nice, done in 9e1ee1f!